pkg/cephmgr/mon/config.go

```diff
@@ -303,6 +304,7 @@ func CreateDefaultCephConfig(context *clusterd.Context, cluster *ClusterInfo, ru
 			OsdObjectStore:         store,
 			RbdDefaultFeatures:     3,
 			CrushtoolPath:          "",
+			FatalSignalHandlers:    "false",
```

Comment:
honestly i haven't yet found it to be very useful (to me) and it adds a bunch of spew to scroll through when post-mortem analyzing the logs.  so we may prefer missing that "recent events" functionality.